integral_view/static/css/timepicker.css

```diff
@@ -0,0 +1,91 @@
+
```

Comment:
We won't need this css if we stick with django's existing widgets which supports date/time input.